config/test.js

```diff
@@ -2,7 +2,7 @@ let config = {};
 
 config.port = 3011;
 config.ws_port = 5667;
-config.iso_img_port = 4202;
+config.iso_img_port = 4201;
```

Comment:
any reasons for that?

Reply:
I used a different port (4202) for testing on purpose to avoid conflicts with dev version running locally.

Reply:
I think I changed it by mistake